docs/starters.yml

```diff
@@ -331,14 +331,17 @@
     - React Icons
 - url: https://parmsang.github.io/gatsby-starter-ecommerce/
   repo: https://github.com/parmsang/gatsby-starter-ecommerce
-  description: no description yet
+  description: Easy to use starter for an e-commerce store
   tags:
     - Styling:Semantic
     - Stripe
     - Moltin
+    - e-commerce
+    - PWA
+    - Authentication
   features:
     - Uses the Moltin eCommerce Api
-    - React 16 (gatsby-plugin-react-next)
+    - React 16 
```

Comment:
not sure you need to add this as a feature. Gatsby v2 is react 16 anyway

Reply:
Agree. I've removed the line.